lib/node_modules/@stdlib/math/base/special/sinh/src/main.c

```diff
@@ -0,0 +1,127 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*
+* ## Notice
+*
+* The original C code, long comment, copyright, license, and constants are from [Cephes]{@link http://www.netlib.org/cephes}. The implementation follows the original, but has been modified for JavaScript.
+*
+* ```text
+* Copyright 1984, 1995, 2000 by Stephen L. Moshier
+*
+* Some software in this archive may be from the book _Methods and Programs for Mathematical Functions_ (Prentice-Hall or Simon & Schuster International, 1989) or from the Cephes Mathematical Library, a commercial product. In either event, it is copyrighted by the author. What you see here may be used freely but it comes with no support or guarantee.
+*
+* Stephen L. Moshier
+* moshier@na-net.ornl.gov
+* ```
+*/
+
+#include "stdlib/math/base/special/sinh.h"
+#include "stdlib/math/base/special/abs.h"
+#include "stdlib/math/base/special/exp.h"
+#include "stdlib/constants/float64/pinf.h"
+#include "stdlib/constants/float64/ninf.h"
+#include "stdlib/constants/float64/ln_two.h"
+#include <stdint.h>
+
+static const double POS_OVERFLOW = 7.09782712893383996843e2 + STDLIB_CONSTANT_FLOAT64_LN2;
```

Comment:
Here, it was not allowing me to declare like Javascript variables 
```c
static const double MAXLOG = 7.09782712893383996843e2;

// ln(2^-1022)
static const double MINLOG = -7.08396418532264106224e2;

static const double POS_OVERFLOW = MAXLOG + STDLIB_CONSTANT_FLOAT64_LN2;
static const double NEG_OVERFLOW = MINLOG - STDLIB_CONSTANT_FLOAT64_LN2;
```
It was throwing an error at `POS_OVERFLOW` declaration, hence I used these magic numbers.

Reply:
That should be fine so long as the values are properly documented.